packages/core/aqua-to-js-compiler/src/utils.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { NonArrowType, ProductType } from '@fluencelabs/interfaces';
+import { readFile } from 'fs/promises';
+
+interface PackageJson {
+    version: string;
+    dependencies: {
+        ['@fluencelabs/aqua-api']: string
+    }
+}
+
+export async function getPackageJsonContent(): Promise<PackageJson> {
+    const content = await readFile('../package.json');
+    const pkg = JSON.parse(content.toString());
+    return pkg;
+}
+
+export function getFuncArgs(domain: ProductType<NonArrowType>): [string, NonArrowType][] {
+    if (domain.tag === 'labeledProduct') {
+        return Object.entries(domain.fields).map(([label, type]) => [label, type]);
+    } else if (domain.tag === 'unlabeledProduct') {
+        return domain.items.map((type, index) => ['arg' + index, type]);
+    } else {
+        return [];
+    }
+}
+
+export function recursiveRenameLaquaProps(obj: unknown): unknown {
+    if (typeof obj !== 'object' || obj === null) return obj;
```

Comment:
```suggestion
    if (obj === null) return null;
```

Reply:
What's the purpose of proposed changes?

Reply:
my bad. Totally misread what's happening here